Apps/Sandcastle/gallery/MSAA.html

```diff
@@ -0,0 +1,177 @@
+          terrainProvider: Cesium.createWorldTerrain(),
+          contextOptions: {
+            requestWebgl2: true,
+          },
+        });
+
+        const scene = viewer.scene;
+        if (!scene.msaaSupported) {
+          window.alert("This browser does not support MSAA.");
+        }
+
+        function createModel(url, height) {
+          const position = Cesium.Cartesian3.fromDegrees(
+            -123.0744619,
+            44.0503706,
+            height
+          );
+          const heading = Cesium.Math.toRadians(135);
+          const pitch = 0;
+          const roll = 0;
+          const hpr = new Cesium.HeadingPitchRoll(heading, pitch, roll);
+          const orientation = Cesium.Transforms.headingPitchRollQuaternion(
+            position,
+            hpr
+          );
+
+          const entity = viewer.entities.add({
+            name: url,
+            position: position,
+            orientation: orientation,
+            model: {
+              uri: url,
+              minimumPixelSize: 128,
+              maximumScale: 20000,
+            },
+          });
+          viewer.trackedEntity = entity;
+        }
+
+        const options = [
+          {
+            text: "Statue of Liberty",
+            onselect: function () {
+              viewer.entities.removeAll();
+              scene.primitives.removeAll();
+              scene.camera.setView({
+                destination: new Cesium.Cartesian3(
+                  1331419.302230775,
+                  -4656681.5022043325,
+                  4136232.6465900405
+                ),
+                orientation: new Cesium.HeadingPitchRoll(
+                  6.032455545102689,
+                  -0.056832496140112765,
+                  6.282360923090216
+                ),
+                endTransform: Cesium.Matrix4.IDENTITY,
+              });
+
+              scene.primitives.add(
+                new Cesium.Cesium3DTileset({
+                  url: Cesium.IonResource.fromAssetId(75343),
+                })
+              );
+            },
+          },
+          {
+            text: "3D Tiles BIM",
+            onselect: function () {
+              viewer.entities.removeAll();
+              scene.primitives.removeAll();
+              const tileset = scene.primitives.add(
+                new Cesium.Cesium3DTileset({
+                  url: Cesium.IonResource.fromAssetId(8564),
+                })
+              );
+              viewer.camera.setView({
+                destination: new Cesium.Cartesian3(
+                  1234138.7804841248,
+                  -5086063.633843134,
+                  3633284.606361642
+                ),
+                orientation: {
+                  heading: 0.4304630387656614,
+                  pitch: -0.16969316864215878,
+                  roll: 6.283184816241989,
+                },
+              });
+            },
+          },
+          {
+            text: "Hot Air Balloon",
```

Comment:
Adjust the default view so that the basket is more visible. It's hard to see the wires at this angle.